### ISSUE TITLE
Conflicting information regarding the debugger with PowerShell in VSCode.

### ISSUE THREAD

**Issue by MarcLaf** [MicrosoftDocs/PowerShell-Docs#10250]
### Prerequisites

- [X] **Existing Issue:** Search the existing issues for this repository. If there is an issue that fits your needs do not file a new one. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "Typo in `Get-Foo` cmdlet" instead of "Typo."
- [X] **Verify Version:** If there is a mismatch between documentation and the behavior on your system, ensure that the version you are using is the same as the documentation. Check this box if they match or the issue you are reporting is not version specific.

### Links

https://learn.microsoft.com/en-us/powershell/scripting/dev-cross-plat/vscode/how-to-replicate-the-ise-experience-in-vscode?view=powershell-7.3#vs-code-tips

### Summary

There is conflicting information regarding the debugger in VS Code and PowerShell. On the page about mimicking the ISE it says the debugger isn't used so you should disable it but the VSCode PS Extension page says that it uses the built in debugger.

https://learn.microsoft.com/en-us/powershell/scripting/dev-cross-plat/vscode/how-to-replicate-the-ise-experience-in-vscode?view=powershell-7.3#vs-code-tips states that the Debugger is not used with PowerShell and if you're doing only PS Scripting that you should disable the debugger. However on https://code.visualstudio.com/docs/languages/powershell#_debugging it says that the PowerShell extension uses the built in debugger in VS Code.

**Comment by sdwheeler** [comment]
This article is referring to the **Debug Console** panel, not the VS Code debugger. See https://code.visualstudio.com/docs/getstarted/userinterface

![image](https://github.com/MicrosoftDocs/PowerShell-Docs/assets/19415881/0a17187c-cebb-493c-8ca4-44cabd5c4a3e)

The **Debug Console** panel is not used by the PowerShell extension. The **Debug Console** is used for debugging VS Code extensions.

But I can see how this could be confusing.

**Comment by Atman-Shastri** [comment]
@sdwheeler 

Sorry for misunderstanding the context of the vs code tip and thank you for taking out time and explaining it. 

I am new to VS code and github and this was my first PR, and as a beginner I got to learn about markdown and references and the process of contributing to the docs through this experience.  
I hope I was able to contribute meaningfully to the docs.

Now that I understand it, I hide the debug console while I debug my powershell scripts in VS code.
